src/utils/network.utils.ts

```diff
@@ -1,9 +1,11 @@
+import mem from 'mem';
+
 import { FILM_TOKEN_METADATA, TEZ_TOKEN_METADATA } from '../token/data/tokens-metadata';
 import { DCP_RPC } from './rpc/rpc-list';
 
-const DCP_NODE_RPC_URLS = [DCP_RPC.url];
+const DCP_NODE_RPC_URL = new URL(DCP_RPC.url).href;
 
-export const isDcpNode = (selectedRpcUrl: string) => DCP_NODE_RPC_URLS.includes(selectedRpcUrl);
+export const isDcpNode = mem((selectedRpcUrl: string) => new URL(selectedRpcUrl).href === DCP_NODE_RPC_URL);
```

Comment:
I don't think we need memoization here - we usually use it for computation-heavy functions, but this one is pretty simple. Memoization in such cases even may add more performance overhead.